crates/meilitool/Cargo.toml

```diff
@@ -16,5 +16,6 @@ file-store = { path = "../file-store" }
 meilisearch-auth = { path = "../meilisearch-auth" }
 meilisearch-types = { path = "../meilisearch-types" }
 serde = { version = "1.0.209", features = ["derive"] }
-time = { version = "0.3.36", features = ["formatting"] }
+time = { version = "0.3.36", features = ["formatting", "parsing", "alloc"] }
 uuid = { version = "1.10.0", features = ["v4"], default-features = false }
+arroy_v04_to_v05 = { package = "arroy", git = "https://github.com/meilisearch/arroy/", rev = "053807bf38dc079f25b003f19fc30fbf3613f6e7" }
```

Comment:
how do we make sure that commits stays alive? Worth replacing with a branch titled `do_not_delete_meilitool_v10_v11` or something?

Reply:
There is a branch, but I think we want to specify both the branch and the rev.
Calling it `do_not_delete` is smart though, I’m going to do it 😂

Reply:
Ah, we can’t specify both the branch and the rev 🤔 
I don’t think specifying only the branch is a good idea though so I made a tag instead.